.github/workflows/ci.yml

```
@@ -119,6 +119,7 @@ jobs:
         env:
           PYTHON: ''
           TRIXI_TEST: ${{ matrix.trixi_test }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # for authenticated downloads without strict rate limits
```
To explain why we need this